src/libs/onlyWeb/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Only run the callback on Web and return the result as Boolean.
```

Comment:
What about Desktop? Also I think it's better to add a more detailed explanation of WHY we're doing this, taking no shortcuts. Maybe we can name this utility something clearer since `onlyWeb` doesn't describe what it's doing

Reply:
Yeah. I put the working logic in but I definitely need help with this. can you suggest a better Naming or technique so that I can set that only for Web or Desktop. There is one more PR, that I am currently working on, has the same situation where we have to disable a prop only for web and desktop. After merging, I will refactor the code to use the generic utility. 

For now, we only need a utility for the web and desktop. Please suggest a good name.